site/index.scroll

```diff
@@ -1,5 +1,7 @@
 import settings.scroll
 
+htmlTitle PLDB - A Programming Languages Database
```

Comment:
but i'm curious, what does this do?

I see the title for the homepage is already `<title>PLDB: A Programming Language Database</title>`, why add this line?

Reply:
ho no, is my mistake. pleas ignore this file, only search.js is the important change for this issue. The file index.scroll change is for other issue. sorry.

Reply:
ok please update this pull request or create a new PR. thanks!